x-pack/test/functional/apps/dashboard/reporting/README.md

```diff
@@ -0,0 +1,17 @@
+## The Reporting Tests
```

Comment:
As this information isn't really dashboard specific, should we but the readme in `functional/reports` instead?

Reply:
> As this information isn't really dashboard specific

For now, this information is only specific to dashboards because it's the only Reporting test that is enabled for this. Other app integrations don't yet have the image comparison test that Dashboard does. When that happens, it might be OK to just copy the file to `visualize/reporting` or `canvas/reporting`.

> should we but the readme in functional/reports instead?

There is no `functional/reports` folder - as part of this PR, we're taking away the notion that Reporting is a standalone app.

Reply:
I meant keeping the folder just for the readme so people can find it more easily, but copying seems even better 👍

Reply:
Thanks for the follow-up :)